### PR TITLE
[v6r13] Several fixes

### PR DESCRIPTION
**Pull request by atsareg**
FIX: DictCache.get() returns now None instead of False if no or expired value
CHANGE: MetaQuery - uses isinstance() to compare types

### CODE REVIEW

File: AccountingSystem/private/DataCache.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def getReportData( self, reportRequest, reportHash, dataFunc ):
     Get report data from cache if exists, else generate it
     """
     reportData = self.__dataCache.get( reportHash )
-    if reportData == False:
+    if reportData is None:
```
Comment (chaen) [inline AccountingSystem/private/DataCache.py]: @sposs  Can you remind me when/where we had this discussion of 'is None' vs 'not' and what was the outcome of it ?

**Comment by chaen** [comment]
Review OK

File: AccountingSystem/private/DataCache.py (hunk 1 above)
Comment (sposs) [inline AccountingSystem/private/DataCache.py]: Nope, I cant remind you, as I don't remember either. It was in a pull request too, for sure though.

Comment (sposs) [inline AccountingSystem/private/DataCache.py]: But the point is that PEP8 advocates the comparison to None if possible. There is a difference between None and 0 for instance. (So now I think it was related to CS values, where 0 would then not be allowed in a specific case)

Comment (atsareg) [inline AccountingSystem/private/DataCache.py]: None is a singleton object in Python. So, "is None" comparison is more explicit.

**Pull request merged**